wd_core.au3

```diff
@@ -1134,6 +1135,10 @@ Func _WD_Option($sOption, $vValue = Default)
 			EndIf
 			$_WD_CONSOLE = $vValue
 
+		Case "consolesufix"

```

Comment:
Same spelling issue here. Also, need to add this option in function header.

Reply:
Take a look on: [small fixes](https://github.com/Danp2/au3WebDriver/pull/242/commits/070aec7f1e04fa8069fdb99340a198018125a5ac)

Reply:
> Also, need to add this option in function header.

And here: [suplemented](https://github.com/Danp2/au3WebDriver/pull/242/commits/3221c678875aa98bd09cd885de21c05a27bb5eee)